docs/app-host/configuration.md

```diff
@@ -0,0 +1,89 @@
+
+```json
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "https": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:17134;http://localhost:15170",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ENVIRONMENT": "Development",
+        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21030",
+        "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22057"
+      }
+    }
+  }
+}
+```
+
+The proceeding launch settings file:
+
+- Has one launch profile called `https`.
+- Configures an .NET Aspire app host project:
+  - The `applicationUrl` property configures the dashboard launch address (`ASPNETCORE_URLS`).
+  - Env vars such as `DOTNET_DASHBOARD_OTLP_ENDPOINT_URL` and `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` are set on the app host.
+
+For more information, see [.NET Aspire and launch profiles](../fundamentals/launch-profiles.md).
+
+> [!NOTE]
+> Configuration described on this page is for .NET Aspire app host project. To configure the standalone dashboard, see [dashboard configuration](../fundamentals/dashboard/configuration.md).
+
+## Common configuration
+
+| Option | Default Value | Description |
+|--|--|--|
+| `ASPIRE_ALLOW_UNSECURED_TRANSPORT` | `false` | Allows communication with the app host without https. `ASPNETCORE_URLS` (dashboard address) and `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` (app host resource service address) must be secured with HTTPS unless true. |
+
+## Resource service
+
+| Option | Default Value | Description |
+|--|--|--|
+| `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` | Automatically generated 128-bit entropy token. | Configures the address of the resource service hosted by the app host. Automatically generated with `launchSettings.json` to have a random port on localhost. For example, `https://localhost:17037`. |
+| `DOTNET_DASHBOARD_RESOURCESERVICE_APIKEY` | `null` | The API key used to authenticate requests made to the app host's resource service. The API key is required if the app host is in run mode, the dashboard isn't disabled, and the dashboard isn't configured to allow anonymous access with `DOTNET_DASHBOARD_UNSECURED_ALLOW_ANONYMOUS`. |
+
+## Dashboard
+
+By default, the dashboard is automatically started by the app host. The dashboard supports [its own set of configuration](../fundamentals/dashboard/configuration.md), and some settings can be configured from the app host.
+
+| Option | Default Value | Description |
+|--|--|--|
+| `ASPNETCORE_URLS` | `null` | Dashboard address. Must be https unless `ASPIRE_ALLOW_UNSECURED_TRANSPORT` or `DistributedApplicationOptions.AllowUnsecuredTransport` is true. Automatically generated with `launchSettings.json` to have a random port on localhost. The value in launch settings is set on the `applicationUrls` property. |
```

Comment:
Alternatively, if not `https` then HTTPS. Also, all references to files should be italicized.

```suggestion
| `ASPNETCORE_URLS` | `null` | Dashboard address. Must be `https` unless `ASPIRE_ALLOW_UNSECURED_TRANSPORT` or `DistributedApplicationOptions.AllowUnsecuredTransport` is true. Automatically generated with _launchSettings.json_ to have a random port on localhost. The value in launch settings is set on the `applicationUrls` property. |
```